apps/frontend/src/libs/components/header/libs/components/header-user-info/header-user-info.tsx

```diff
@@ -0,0 +1,32 @@
+import defaultAvatar from "~/assets/img/default-avatar.png";
+import { AppRoute } from "~/libs/enums/enums.js";
+import { useAppNavigate, useCallback } from "~/libs/hooks/hooks.js";
+import { type UserDto } from "~/modules/users/users.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	user: UserDto;
+};
+
+const HeaderUserInfo: React.FC<Properties> = ({ user }: Properties) => {
+	const navigate = useAppNavigate();
+
+	const handleClick = useCallback(() => {
+		navigate(AppRoute.PROFILE);
+	}, [navigate]);
+
+	return (
+		<label className={styles["container"]}>
+			<input className="visually-hidden" onClick={handleClick} type="button" />
```

Comment:
Why do we need input here?

Reply:
To make the container clickable. Is there an alternative approach?

Reply:
It is not correct to have input for click event. It should be a link/button